WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalAdvancedStatsPage.js

```diff
@@ -427,58 +429,69 @@ ZaGlobalAdvancedStatsPage.plotChart = function (id, fields, colDef, newData) {
             animation: false,
             responsive: true,
             maintainAspectRatio: false,
-            legend: {
-                display: true,
-                position: 'bottom'
+            datasets: {
+                line: {
+                    tension: 0.4
+                }
             },
-            tooltips: {
-                mode: 'point',
-                custom: function(tooltip) {
-                    if (!tooltip) return;
-                    // disable displaying the color box;
-                    tooltip.displayColors = false;
+            plugins: {
+                legend: {
+                    display: true,
+                    position: 'bottom'
                 },
-                callbacks: {
-                    title: function() {
-                        return null;
+                tooltip: {
+                    enabled: true,
+                    mode: 'point',
+                    external: function(tooltip) {
+                        if (!tooltip) return;
+                        // disable displaying the color box;
+                        tooltip.displayColors = false;
                     },
-                    label: function (tooltipItems, data) {
-                        return ZaGlobalAdvancedStatsPage.getDataTipText(tooltipItems, undefined, data.datasets[tooltipItems.datasetIndex].label);
+                    callbacks: {
+                        title: function() {
+                            return null;
+                        },
+                        label: function (context) {
+                            return ZaGlobalAdvancedStatsPage.getDataTipText(context, undefined, context.dataset.label);
```

Comment:
`ZaGlobalAdvancedStatsPage.getDataTipText` is defined as follows
```
(at line 82)
ZaGlobalAdvancedStatsPage.getDataTipText = function (item) {
```
The passing parameters `(context, undefined, context.dataset.label)` does not match the definition. Is `context` only used?

Reply:
Yes. With the new changes relating to v4 update, only `context` is needed by `getDataTipText`.
Fixed. Thanks.